### PR TITLE
bugfix/gosh:

### PR DESCRIPTION
**Pull request by ChriMarMe**
* Remove runInteractive path, -tabcomp flag, unify to run with tab completion only.
* Remove runErr and related code. Shell shall not terminate, when a runErr occurs.

Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: cmds/exp/gosh/gosh.go [hunk 1]
```diff
@@ -162,13 +112,13 @@ func (s shell) runInteractiveTabCompletion(r *interp.Runner, stdout io.Writer) e
 				return true
 			}
 
-			runErr = r.Run(context.Background(), stmt)
```
Comment (MDr164) [inline cmds/exp/gosh/gosh.go:165]: Should we maybe print out any occurence of runErr instead of ignoring it here?

**Review by MDr164 — CHANGES_REQUESTED** [review]
As you removed a method from the gosh code, you'll also have to remove the corresponding test or the CI will fail.

File: cmds/exp/gosh/gosh.go (hunk 1 above)
Comment (ChriMarMe) [reply cmds/exp/gosh/gosh.go:165]: Printing this error would result in error code 127 after EVERY use of non existing commands.

Comment (MDr164) [reply cmds/exp/gosh/gosh.go:165]: Oh yes, you're right!

**Review by MDr164 — APPROVED** [review]

**Pull request merged**